src/Mgate/SuiviBundle/Entity/Etude.php

```diff
@@ -344,7 +344,10 @@ public function getFs()
 
     public function getNumero()
     {
-        return $this->mandat * 100 + $this->num;
+        //return $this->mandat * 100 + $this->num;
```

Comment:
Working code should not be commented (sensiolabs insight rules for platinum medal).

Can you please remove that line  ?